integrations/airflow/marquez_airflow/extractors/sql/parser.py

```diff
@@ -74,8 +88,13 @@ class SqlParser:
     This class parses a SQL statement.
     """
 
-    @staticmethod
-    def parse(sql: str) -> SqlMeta:
+    def __init__(self):
+        self.ctes = set()
+        self.intables = set()
+        self.outtables = set()
+
+    @classmethod
+    def parse(cls, sql: str) -> SqlMeta:
```

Comment:
We should pass the default schema as a parameter to enable normalizing table names

Reply:
will do in following PR :+1: